deluge/plugins/Scheduler/deluge/plugins/scheduler/gtkui.py

```diff
@@ -13,26 +13,30 @@
 
 import logging
 
-import gtk
+from gi.repository import Gtk, Gdk
 
 import deluge.component as component
 from deluge.plugins.pluginbase import GtkPluginBase
 from deluge.ui.client import client
+from gi import pygtkcompat
```

Comment:
fixed and yes theres still something(cant recall right now) that isnt updated for the api change fairly small change but still .. needs TODO